bundles/storefront-search.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
+services:
+  app-search:
+    image: mslabko/php-with-grpc
+# to re-build image with gRPC use the following docker file:
```

Comment:
We don't need to build it each time, please comment / remove "build" section 
Also, please pull latest changes and replace image with "rkostiv/php-with-rr-grpc" (it will go with pre-build grpc server )

Reply:
replaced with rkostiv/php-with-rr-grpc